File: lab/c/matematica_e_operadores/sucessor_e_antecessor.c
#include <stdio.h>


int sucessor_e_antecessor(int numero){
    int sucessor = numero + 1;
    int antecessor = numero - 1;

    printf("Antecessor: %d\n", antecessor);
    printf("Sucessor: %d\n", sucessor);
}

int main(){
    int numero;
    scanf("%d", &numero);

    sucessor_e_antecessor(numero);
}

File: lab/javascript/matematica_e_operadores/sucessor_e_antecessor.js
function sucessor_e_antecessor(numero){
    let sucessor = numero + 1;
    let antecessor = numero - 1;

    console.log(`Antecessor: ${antecessor}`);
    console.log(`Sucessor: ${sucessor}`);
}


const prompt = require('prompt-sync')();
let numero = Number(prompt());
sucessor_e_antecessor(numero);
